test/expect/TestFXAPIBackwardCompatibility.test_class_member_back_compat-fx_backcompat_class_members.expect

```diff
@@ -15,5 +15,4 @@ torch.fx.proxy.Attribute ['node']
 torch.fx.proxy.GraphAppendingTracer []
 torch.fx.proxy.Proxy ['keys']
 torch.fx.proxy.TraceError []
-torch.fx.proxy.TracerBase ['check_mutable_operations', 'create_arg', 'create_node', 'create_proxy', 'iter', 'keys', 'proxy', 'proxy_buffer_attributes', 'record_stack_traces', 'to_bool', 'trace_asserts', 'traced_func_name']
-torch.fx.subgraph_rewriter.Match ['anchor', 'nodes_map']
```

Comment:
As discussed offline, this PR is making a BC-breaking change without going through the proper deprecation process and is flagrantly bypassing BC enforcement mechanisms. Please do not merge this change without changes to make it BC

Reply:
Updated per request.